### PR TITLE
Re-introduce 29593: pipeline configuration for required copyright header checking

### PR DESCRIPTION
**Pull request by wraschke**
- [x] I have considered the risk of behavior change or other zero migration impact (https://github.com/OpenLiberty/open-liberty/wiki/Behavior-Changes).
- [x] If this PR fixes an Issue, the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN" (verify `release bug` label if applicable: https://github.com/OpenLiberty/open-liberty/wiki/Open-Liberty-Conventions).
- [x] If this PR resolves an external Known Issue (including APARS), the description includes "Fixes #FILLMEIN" or "Resolves #FILLMEIN".

### CODE REVIEW

**Comment by wraschke** [comment]
!liberty@dev

**Comment by wraschke** [comment]
!libby@dev

**Comment by wraschke** [comment]
!libby@dev

File: .ci-orchestrator/checks.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  triggerRank: 50
+  groups: ["LibertyDev"]
+  keyword: "!libby"
+  aliasKeywords:
+    - "#run-libby-bot"
+    - "#run-libby"
+    - "#libby"
+  propertyDefinitions:
+    - name: githubPRNumber
+      defaultValue: ${github_pr_number}
```
Comment (fritze2) [inline .ci-orchestrator/checks.yml:18]: Should have
```
      steps:
      - stepName: Copyright Header Check
      - stepName: Prohibited Terms Check
```

File: .ci-orchestrator/checks.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+- type: manual
+  triggerName: "libby-manual"
+  description: "Runs CI Orchestrator-managed Jenkins jobs to examine pull request changes and create required status checks"
+  triggerRank: 50
+  groups: ["LibertyDev"]
+  propertyDefinitions:
+    - name: githubPRNumber
+      isRequired: true
+      defaultValue: "Number of Open Liberty pull request to verify"
+      description: "Number of Open Liberty pull request to verify"
```
Comment (fritze2) [inline .ci-orchestrator/checks.yml:31]: Should have
```
      steps:
      - stepName: Copyright Header Check
      - stepName: Prohibited Terms Check
```

**Review by fritze2 — DISMISSED** [review]
Minor changes suggested, but not necessary.

**Review by fritze2 — DISMISSED** [review]

**Comment by wraschke** [comment]
!libby@dev

**Comment by wraschke** [comment]
!libby@dev

**Review by fritze2 — APPROVED** [review]

**Comment by wraschke** [comment]
!libby

**Pull request merged**